tokens.xml

```diff
@@ -517,7 +517,6 @@ Creature tokens you control have flying and vigilance.</text>
             <set picURL="https://img.scryfall.com/cards/large/front/3/f/3fc3a29a-280d-4f2c-9a01-8cfead75f583.jpg">EVE</set>
             <set picURL="https://img.scryfall.com/cards/large/front/c/3/c3cbd01b-dbf5-424a-855d-f9a9d7e7e414.jpg">LRW</set>
             <set picURL="https://img.scryfall.com/cards/large/front/d/0/d02d5670-fff3-4011-8e45-e6843e4e8988.jpg">DST</set>
-            <reverse-related>"Beast Without"</reverse-related>
```

Comment:
is this change significant?

Reply:
The "Beast Without" entry should be removed. It was a leftover from scryfalls spoiler name for Sawtusk Demolisher.